pom.xml

```diff
@@ -116,6 +116,14 @@
         <module>presto-benchto-benchmarks</module>
     </modules>
 
+    <repositories>
```

Comment:
I believe the required Airlift changes have been merged and released.  If so, can you remove this commit.  If not, please link to the Airlift PR so I can get it merged.